scripts/generate_config.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3
+# type: ignore
+
+"""Mbed TLS user configuration file generate.
```

Comment:
Please document what this script does.

Without knowing what this is supposed to do, it's impossible to review.